app/scripts/controllers/main.js

```diff
@@ -3,13 +3,17 @@
 /**
  * Main App Controller -- Manage all emails visible in the list
  */
+var refreshTimeout = null
+var notificationTimeout = null
 
 app.controller('MainCtrl', [
   '$scope', '$rootScope', '$http', 'Email', '$route', '$location', 'Favicon',
   function ($scope, $rootScope, $http, Email, $route, $location, Favicon) {
     $scope.items = []
     $scope.configOpen = false
     $scope.currentItemId = null
+    $scope.notificationsSupported = 'Notification' in window
```

Comment:
I think we should give user feedback if it's supported also using `isSecureContext`.

Reply:
Interesting, what kind of feedback are you thinking of?

Reply:
I decided to add a `title` attribute on the Notifications toggle now. Also with the new toggle design, Notifications is now greyed out if the browser context does not support it:

<img width="510" alt="Screen Shot 2020-03-30 at 9 18 57 AM" src="https://user-images.githubusercontent.com/1509457/77916766-9aa62880-7267-11ea-979a-2c1d26cd3bd6.png">